client/cli/ok.py

```diff
@@ -119,7 +119,7 @@ def main():
     # Instantiating assignment
     try:
         assign = assignment.load_config(args.config, args)
-    except (ex.LoadingException, ex.SerializeException) as e:
+    except ex.OkException as e:
```

Comment:
Instead of leaving this as "OkException", should we make Loading and Serialize exceptions subclasses of another class? It's kind of confusing having this as an "OkException" since that's the most general exception that we have.